### ISSUE TITLE
Safari Extension UI issues with Proxmox TOTP entry

### ISSUE THREAD

**Issue by rh8056** [bitwarden/clients#12085]
### Steps To Reproduce

When using the web interface to Proxmox in Safari, the Bitwarden extension causes issues with the TOTP entry screen.
 
If the Bitwarden extension is disabled, when the TOTP window pops up, the cursor is defaulted into the entry box and ready to accept a keystroke.

### Expected Result

The TOTP window would pop up and allow immediate entry into the TOTP code field

### Actual Result

The TOTP window pops up, but interaction with it is disabled (I cannot type, I cannot click into the TOTP entry box, I cannot tab to the TOTP entry box). 

Click somewhere in the window, then hit tab a few times to get into the TOTP entry, then enter the key. 

### Operating System

macOS

### Web Browser

Safari

### Build Version

Version: 2024.10.1

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Neonwarden** [comment]
Hi there,

This issue has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by wrobelda** [comment]
This issue is still here, I just got burned by it. Notably this does not happen in Safari nor in Chrome for me, but it in Brave.

**Comment by RonaldPM** [comment]
Still an issue in 2025 July. Happens with all Proxmox web apps such as PVE, PBS etc.

**Comment by wrobelda** [comment]
There's a chance this could an issue on Proxmox end

**Comment by RonaldPM** [comment]
I don't believe so. It seem to work fine without the extension on incognito. I had also raised an issue with Proxmox [here](https://bugzilla.proxmox.com/show_bug.cgi?id=6492)

**Comment by niktss** [comment]
Also happens for me in Firefox. Workaround: Bitwarden settings --> Auto-fill --> Uncheck "Show autofill suggestions on form fields"

**Comment by aism** [comment]
Same issue on Brave.

**Comment by TheDuck1234** [comment]
same issue in chrome, have to disable bitwarden extension before im able to type in the 2FA

**Comment by RonaldPM** [comment]
@TheDuck1234 or you can right click on the MFA popup and delete the div with id/class called `presentation`

**Comment by zaxlofful** [comment]
This also happens in Firefox on Windows.

### My Workaround:
_**Use the tab key, until the cursor finds its way to the MFA text box. Then you can enter your code.**
This also works for the other pop-ups the extension breaks in the Proxmox GUI, you can tab to the non-subscription acceptance button._
